sdk/keyvault/azure-keyvault-certificates/samples/backup_restore_operations.py

```diff
@@ -61,7 +61,7 @@
     # The storage account certificate is no longer in use, so you can delete it.
     client.delete_certificate(name=cert_name)
     # To ensure certificate is deleted on the server side.
-    time.sleep(30)
+    time.sleep(20)
```

Comment:
I think there should be a print statement like "\n.. Deleting Certificate", sorry this is my bad